schema/project-level/project-schema.json

```diff
@@ -684,8 +702,7 @@
                 "type": [
                   "string"
                 ]
-              },
-              "uniqueItems": true
```

Comment:
I think OC4IDS had opted-in to more strict validation here.

Reply:
Ah yes, my mistake. Fixed in https://github.com/open-contracting/infrastructure/pull/282/commits/683ef4da0c2d20dfc26e94784e7db0d2fcfeec61